notebooks/GWAS_1000Genomes_p1/main.py

```diff
@@ -0,0 +1,143 @@
+# Notebook author: Beth Sheets
+# Herzog version: Ash O'Farrell
+
+# The notebook didn't have a markdown title at the time but I've dubbed it Phenotype Consolidator.
+# The name of the actual file is 1-Prepare-Gen3-Data-For-Exploration
+
+import os
+import herzog
+from unittest import mock
+
+# This notebook has a dependency on terra_data_table_util that, for now, is being mocked
+consolidate_gen3_pheno_tables = mock.MagicMock()  # test fixture
+
+# Mock the environment
+os.environ['WORKSPACE_NAME'] = "cicd-tester-1000genomes-gwas"
+os.environ['WORKSPACE_BUCKET'] = "gs://fc-eb68164b-bae8-4892-83b8-637c1385b09a"
+os.environ['GOOGLE_PROJECT'] = "firecloud-cgl"
+
+with herzog.Cell("markdown"):
+    """
+    # Phenotype Consolidater
+    *version: 2.0.1*
+
+    ## Notebook Overview
+
+    This notebook consolidates the phentotypic data in tables imported to Terra from Gen3 into one consolidated table of metadata with familiar subject IDs for downstream analysis. It uses functions defined in the terra_data_table_util notebook, and can be adapted to consolidate different metadata.
```

Comment:
Can we link terra-data-table-util text to this terra.biodatacatalyst.nhlbi.nih.gov/#workspaces/biodata-catalyst/BioData%20Catalyst%20Collection/notebooks/launch/terra_data_table_util.ipynb?

Add "Note: `terra-data-table-util` needs to be in the same workspace as this notebook to run successfully."